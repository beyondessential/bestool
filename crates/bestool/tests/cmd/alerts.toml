bin.name = "bestool"
args = "tamanu --root ./tamanu alerts --interval 1w --dir ./alerts --dry-run"
