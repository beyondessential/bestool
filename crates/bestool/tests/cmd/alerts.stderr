 WARN bestool::args: logging configured from RUST_LOG RUST_LOG=warn
 WARN bestool::args: âš  RUST_LOG environment variable set or hardcoded, logging options have no effect
